src/routes.rs

```diff
@@ -19,12 +19,14 @@ use tracing::{error, info};
 lazy_static::lazy_static! {
     static ref PAGE: String = {
         let template = include_str!(concat!(env!("OUT_DIR"), "/index.html"));
+        let server_config = Config::default();
```

Comment:
Hmm this is going to cause the node_url to always be localhost, even in production. Is logging the node url really necessary?

Reply:
We use it to show the node URL and also to create the block-explorer link. It's possible to get direct from the env, what you think?

Reply:
Hmm we won't be able to use lazy_static anymore then. We should move all of this logic here under a function which takes the server_config as an arg, and then use a library like [`memoize`](https://github.com/dermesser/memoize) to cache it for all future requests.